src/components/Admin/custom/PostPreview.tsx

```diff
@@ -1,22 +1,20 @@
 import {Stack} from '@mui/material'
 import {ThemeProvider} from '@mui/material/styles'
 import Grid from '@mui/material/Unstable_Grid2'
-import {FC, useState} from 'react'
+import {FC} from 'react'
 import {FormDataConsumer} from 'react-admin'
 
 import {Post} from '@/components/Posts/Post'
-import {PostDetail} from '@/components/Posts/PostDetail'
 import {theme} from '@/theme'
 
 export const PostPreview: FC = () => {
-  const [isDetailOpen, openDetail] = useState<boolean>(false)
   return (
     <ThemeProvider theme={theme}>
       <FormDataConsumer>
         {({formData}) => (
           <Grid container columnSpacing={5} sx={{width: '100%'}}>
-            <Grid xs={4}>
-              <Stack gap={5}>
+            <Grid xs={12} md={4}>
+              <Stack gap={5} sx={{background: 'white', ml: '-10px', p: '10px', color: 'black'}}>
```

Comment:
toto nejak matchuje paddingy Postov? vyzera to jak nejaky zbytocny hack, ale co ja viem

Reply:
no buttons maju `ml: - 10px`, takze ich background isiel mimo toho, zaroven pre dark mode to vyzeralo zvlastne - hej je to taky divny hack ale meh

kazdopadne ako to pozeram, staci tam ten padding (a popravde na kazdej obrazovke je to inak siroke, takze vlastne sa neda spoliehat na to co sa tam zobrazi)